blazor-features/ErrorBoundaries/Tests/ErrorBoundariesTests.cs

```diff
@@ -0,0 +1,63 @@
+using Bunit;
+using ErrorBoundaries.Pages;
+using Xunit;
+
+namespace Tests;
+public class ErrorBoundariesTests : TestContext
+{
+    [Fact]
+    public void GivenAValidContext_WhenComponentIsRenderedForFirstTime_ThenRendersCounterComponent()
+    {
+        // Act
+        var cut = RenderComponent<Index>();
+
+        // Assert
+        var counter = cut.FindComponent<Counter>();
+        Assert.NotNull(counter);
+    }
+
+    [Fact]
+    public void GivenAValidContext_WhenCounterExceedsFive_ThenErrorIsShown()
+	{
+        // Arrange
+        const int count = 4;
+
+        // Act
+        var cut = RenderComponent<Index>();
+        var counter = cut.FindComponent<Counter>();
+        var counterButton = counter.Find("button");
+
+		for (var i = 0; i <= count; i++)
+		{
+            counterButton.Click();
+		}
```

Comment:
If I had to guess this might be a mix of tabs and spaces which GitHub doesn't show properly. Try to use either one of them.

Reply:
Weird.. that wasn't showing up locally for me. Should be fixed now.

Reply:
Happens more than you know.
I use the option in VS (CTRL + R , CTRL + W) that shows tabs/spaces visually:
![image](https://user-images.githubusercontent.com/29179238/182020107-df53cd0a-f3c5-4d6d-ad5f-5b609023fe08.png)
It helps a lot.

Reply:
I guess it's not that big of a deal, but we should try to keep to code consistent as much as we can.